web-calc.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <title>Calculator</title>
+        <script src="index.js" type="text/javascript"></script>
+    </head>
+    <div class="calculator">
+    <body>
+        <h1>"Quick Maths" Calculator</h1>
+        <table border="2" width="450" cellpadding="2" cellspacing="2">
+    <tr>
+        <td>First Number</td>
+        <td>Second Number</td>
+        <td>Result</td>
+    </tr>
+    <tr>
+        <td><input name="number1" type="text" size=10 id='num1'/></td>
+        <td><input name="number2" type="text" size=10 id='num2'/></td>
+        <td> <input type="text" id='result' readonly ></td>
+    </tr>
+    <td colspan="2">
+        <button onclick="showresult('1')">+</button>
+        <button onclick="showresult('2')">-</button>
+        <button onclick="showresult('3')">*</button>
+        <button onclick="showresult('4')">/</button>
```

Comment:
Rather than 1,2,3,4 using something like '+', "-', '*', '/' provides a lot more context to the person reading this which may very well be yourself 6 months later. I guarantee if you just looked at the JavaScript in 6 months your reaction would be

![image](https://encrypted-tbn0.gstatic.com/images?q=tbn%3AANd9GcTCZpEQodp8TAYEVWTEbnRTGqD5deWEVLmEK5kMcO7NIo0DCN50)